shell/ci/testing/setup-devenv.sh

```diff
@@ -96,3 +97,8 @@ if [[ $E2E == "true" ]]; then
   info "Starting E2E test runner"
   TEST_TAGS=or_test,or_e2e exec "$("$DIR/../../gobin.sh" -p "github.com/getoutreach/devbase/v2/e2e@$(cat "$DIR/../../../.version")")"
 fi
+
+if [[ $CHECK_CONFIG == "true" ]]; then
```

Comment:
Instead of putting this here, you should put this into the `shell/linters` directory and use the linter framework there